### ISSUE TITLE
Lot's of notifications because sever failed

### ISSUE THREAD

**Issue by Barbapapazes** [vuejs/language-tools#2703]
Hello,

Recently (less than a week), I've got a lot of failed with Volar but I don't know where does it come from.

```
[Error - 2:45:09 PM] Server initialization failed.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:09 PM] Vue Syntactic Server client: couldn't create connection to server.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Info  - 2:45:11 PM] Connection to server got closed. Server will restart.
true
[Error - 2:45:12 PM] Server initialization failed.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:12 PM] Vue Syntactic Server client: couldn't create connection to server.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:12 PM] Restarting server failed
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Info  - 2:45:14 PM] Connection to server got closed. Server will restart.
true
[Error - 2:45:14 PM] Server initialization failed.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:14 PM] Vue Syntactic Server client: couldn't create connection to server.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:14 PM] Restarting server failed
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Info  - 2:45:16 PM] Connection to server got closed. Server will restart.
true
[Error - 2:45:17 PM] Server initialization failed.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:17 PM] Vue Syntactic Server client: couldn't create connection to server.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:17 PM] Restarting server failed
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Info  - 2:45:19 PM] Connection to server got closed. Server will restart.
true
[Error - 2:45:19 PM] Server initialization failed.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:19 PM] Vue Syntactic Server client: couldn't create connection to server.
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:19 PM] Restarting server failed
  Message: Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in c:/Users/esoubiran/.vscode/extensions/ms-vscode.vscode-typescript-next-5.1.20230423/extensions/node_modules/typescript/lib
  Code: -32603 
[Error - 2:45:22 PM] The Vue Syntactic Server server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.
```

**Comment by Barbapapazes** [comment]
seems related to https://github.com/vuejs/language-tools/issues/2663

**Comment by Barbapapazes** [comment]
Seems to be fixed with 1.4.4!
